src/clj/game/cards/resources.clj

```diff
@@ -1139,6 +1139,17 @@
                  :effect (effect (derez target)
                                  (trash card {:cause :ability-cost}))}]}
 
+   "Miss Bones"
+   {:data {:counter {:credit 12}}
+    :implementation "Credit use restriction not enforced"
+    :abilities [{:counter-cost [:credit 1]
+                 :msg "gain 1 [Credits] for trashing installed cards"
+                 :async true
+                 :effect (req (gain-credits state :runner 1)
```

Comment:
This shouldn't use gain right since it doesn't technically go to runner credit pool right?

Reply:
I don't think we have a way to denote that credits are only for a specific usage. `Scrubber` adds them to the credit pool.

Reply:
We have `take-credits` for this very use case.

Reply:
Yeah, I was more thinking about the extra `runner-gain-credit` events that `gain-credits` fires off that should be avoided.